com.woltlab.wcf/templates/shared_multiplePagesSelectionFormField.tpl

```diff
@@ -0,0 +1,19 @@
+{include file="shared_multipleSelectionFormField"}
+
+{if $field->getVisibleEverywhereFieldId() !== null}
+	<script data-relocate="true">
+		const label = document.querySelector('label[for="{$field->getPrefixedId()}"]');
+
+		document.querySelectorAll('input[name="{$field->getVisibleEverywhereFieldId()}"]').forEach((input) => {
+			input.addEventListener("change", () => {
+				setLabelText(input.value);
+			});
+		});
+
+		function setLabelText (value) {
+			label.innerHTML = parseInt(value) === 0 ? '{unsafe:$field->getLabel()|encodeJS}' : '{unsafe:$field->getInvertedLabel()|encodeJS}';
+		}
+
+		setLabelText(document.querySelector('input[name="{$field->getVisibleEverywhereFieldId()}"]:checked').value);
```

Comment:
```suggestion
		{
			const label = document.querySelector('label[for="{$field->getPrefixedId()}"]');

			document.querySelectorAll('input[name="{$field->getVisibleEverywhereFieldId()}"]').forEach((input) => {
				input.addEventListener("change", () => {
					setLabelText(input.value);
				});
			});

			function setLabelText (value) {
				label.innerHTML = parseInt(value) === 0 ? '{unsafe:$field->getLabel()|encodeJS}' : '{unsafe:$field->getInvertedLabel()|encodeJS}';
			}

			setLabelText(document.querySelector('input[name="{$field->getVisibleEverywhereFieldId()}"]:checked').value);
		}
```

This prevents the variable and function from polluting the global namespace by scoping it.